test/test/cpu/CpuTests.java

```diff
@@ -32,4 +35,42 @@ public void itimerTotal(TestProcess p) throws Exception {
         Output out = p.profile("-d 2 -e itimer -i 100ms --total -o collapsed");
         assertCloseTo(out.total(), 2_000_000_000, "itimer total should match profiling duration");
     }
+
+    @Test(mainClass = CpuBurner.class, os = Os.LINUX, cpu = 1)
+    public void perfEventsWrongTargetCpu(TestProcess p) throws Exception {
+        Output out = p.profile("-d 2 -e cpu -i 100ms --total -o collapsed --target-cpu 2");
+        assertCloseTo(out.total(), 0, "perf_events total should be 0 when the wrong CPU is targeted");
```

Comment:
Not "close to", but it should be strictly zero in this case, right?

Reply:
Yes indeed: bb909cd085144d9d49124f2181832eb150e7c4ec